core/src/views/header.rs

```diff
@@ -112,4 +112,15 @@ impl<'a> HeaderView<'a> {
         let seal = self.seal();
         seal.into_iter().map(|s| rlp::Rlp::new(&s).data().map(|x| x.to_vec())).collect()
     }
+
+    pub fn view(&self) -> Option<u64> {
+        let seal = self.seal();
+        let rlp_view = seal.get(1);
```

Comment:
You can utilize the `?` operator for `Option` type.
let rlp_view = seal.get(1)?; then `rlp_view` don't need to be manually unwrapped.